test/csuite/wt2447_join_main_table/main.c

```diff
@@ -115,6 +115,8 @@ main(int argc, char *argv[])
     case TABLE_ROW:
         printf("Table type: rows\n");
         break;
+    case TABLE_NOT_SET:
+        testutil_die(ENOTSUP, "Unknown table type (%d)\n", (int)opts->table_type);
```

Comment:
Alphabetize (probably). I'd normally push hard for this, but it's also the error case so might feel right at the end.

Reply:
Fixed.